app/views/shared/authentication/_providers_sidebar.html.erb

```diff
@@ -1,9 +1,16 @@
-<% authentication_enabled_providers.each do |provider| %>
-  <a
-    href="<%= provider.sign_in_path(state: "navbar_basic") %>"
-    class="crayons-btn crayons-btn--l crayons-btn--brand-<%= provider.provider_name %> crayons-btn--icon-left"
-    data-no-instant>
-    <%= inline_svg_tag("#{provider.provider_name}.svg", aria: true, class: "crayons-icon", title: provider.provider_name) %>
-    Sign In with <%= provider.official_name %>
-  </a>
-<% end %>
+<div class="border-0 border-b-1 border-solid border-base-20 p-2 grid gap-2">
+  <% if any_selfserve_auth? %>
+    <%= render partial: "shared/authentication/providers_nav_menu" %>
+    <a
+      href="/enter"
```

Comment:
**nitpick:** We're attempting to use path helpers as much as possible, so let's please do this here too.

```suggestion
      href="<%= sign_up_path %>"
```